distribution/scenes/textures/patterns/agate.pov

```diff
@@ -2,35 +2,38 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Agate pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Agate pattern example agate.pov.
+//
 // -w800 -h600 +a0.3
 
 #version 3.7;
-
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
 
-#include "colors.inc"
+#declare Black = srgb <0,0,0>;
+#declare White = srgb <1,1,1>;
 
-#declare T1=
- texture{
-   pigment{
-     agate color_map{[0.0 Black][1.0 White]}
-     scale 0.5
-   }
- }
+#declare T1 = texture {
+    pigment {
+        agate
+        color_map {
+            [0.0 Black]
+            [1.0 White]
+        }
+        scale 0.5
+    }
+}
```

Comment:
Not opposed to further change. I picked a standard close to our C++ code base for indentation & what I use personally as a standard for these updates given there were previously many in this directory - let alone across all the scene and include files. 

Also something which didn't cram large amounts of 'function' onto one line - again thinking of users new to POV-Ray. I don't feel 'short' code should be our aim when instructing.